Resources/Prototypes/Recipes/Lathes/medical.yml

```diff
@@ -222,4 +222,4 @@
   completetime: 2
   materials:
     Steel: 100
-    Plastic: 100
```

Comment:
Random newline deletion. Revert with git or re-save with an editor that supports the [editorconfig spec](https://editorconfig.org/).